### ISSUE TITLE
React won't pick up state object from .Net if field has an attribute

### ISSUE THREAD

**Issue by ibuildit** [dsuryd/dotNetify-Elements#22]
If I add attribute to a "state object" field (what is Observed), then it won't show up in the activity feed. If I remove the attributes, it works fine. Is there a way we could make it work even with attributes?

thanks!

    public class Activity {

        [JsonProperty("ID")]
        public int Id { get; set; }

        [JsonProperty("personName")]
        public string PersonName { get; set; }
    }

**Comment by dsuryd** [comment]
Since you're renaming properties that get sent to the client, you need to make sure the client code uses the new name.  Check console log errors for places you need to update.

**Comment by ibuildit** [comment]
Ah! I failed to think about that json is involved between server and client.

[JsonProperty("PersonName")]
public string PersonName { get; set; }

This works perfectly. Thanks for clearing that up.

On a sidenote, adding MVC Web API capabilities to dotnetify and exposing API endpoints made it incredibly useful as a dashboard.

All I did was add:
"services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);"
to ConfigureServices()

and add
app.UseMvc();
to Configure()

and dependencies using Microsoft.AspNetCore.Mvc;

After that I just added a new controller in VS and kazaaam! I have API endpoint!
